### PR TITLE
Make docs build conditional

### PR DESCRIPTION
**Pull request by palexster**
# Description

This PR makes the trigger for the documentation pipeline conditionals to effective modifications of the docs directory. This should avoid several useless builds.

# How Has This Been Tested?

- [x] E2E Tests

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -134,32 +134,52 @@ jobs:
...
                "commit": "${{ needs.configure.outputs.commit_ref }}",
                "repo-name": "${{ needs.configure.outputs.repo-name }}"
              }
+
+  docs-pipeline-trigger:
+    runs-on: ubuntu-latest
+    needs: configure
+    if: needs.configure.outputs.ok-to-continue == 'true' && github.event.pull_request.draft == false
+    steps:
+      - name: Check if /operators files changed
```
Comment (giorio94) [inline .github/workflows/integration.yml:162]:
```suggestion
      - name: Check if /docs files changed
```

**Review by giorio94 — APPROVED** [review]
/lgtm

**Review by aleoli — APPROVED** [review]

**Pull request merged**